### PR TITLE
Manis work

### PR DESCRIPTION
**Pull request by ManiAkasapu**
Font file added.

### CODE REVIEW

**Comment by matt-hu** [comment]
@ManiAkasapu In future make sure your pull requests target `next-release`, not `main`

**Comment by ManiAkasapu** [comment]
Oh! You have already changed the base branch.. Thanks. I may have overlooked it.

**Comment by matt-hu** [comment]
@ManiAkasapu I like the look of what you've done, but currently it will remove lots of the work @dexslab has done to add custom themes. I'm not entirely sure the best way to rectify this.

File: src/App.vue [hunk 1]
```diff
@@ -1,138 +1,54 @@
...
+          value="about"
+          @click="router.push('/about'); drawer = !drawer"
+        />
       </v-list>
     </v-navigation-drawer>
     <v-main>
-      <router-view/>
+      <RouterView/>
     </v-main>
-    <v-footer height="25">
```
Comment (matt-hu) [inline src/App.vue:50]: This section here has been removed, but this is the theme selector switch

**Review by matt-hu — CHANGES_REQUESTED** [review]
I've added a comment to show where the theme selector has been removed

File: src/App.vue (hunk 1 above)
Comment (matt-hu) [reply src/App.vue:50]: I've just noticed that #8 does remove some of that so I'll have another look once that's merged

**Comment by ManiAkasapu** [comment]
I think something wrong happened..  I have not edited those..  When I compare, it is same as next-release.. I have to look into it.. 
Thanks looking.. Can you close or reject this PR..

**Comment by dexslab** [comment]
Please merge the changes from next-release i finally closed my PR

**Pull request closed without merging**